channel.go

```diff
@@ -269,6 +286,11 @@ func NewChannel(serviceName string, opts *ChannelOptions) (*Channel, error) {
 	if opts.RelayHost != nil {
 		opts.RelayHost.SetChannel(ch)
 	}
+
+	// Start the idle connection timer.
+	ch.mutable.idleSweep = newIdleSweep(ch, opts)
+	ch.mutable.idleSweep.Start()
```

Comment:
no point separating the new and start, we can simplify by just having `startIdleSweep` and then we don't have the double-start problem

Reply:
Done